src/fake_useragent/__init__.py

```diff
@@ -2,4 +2,4 @@
 
 from fake_useragent.errors import FakeUserAgentError, UserAgentError
 from fake_useragent.fake import FakeUserAgent, UserAgent
-from fake_useragent.settings import __version__ as VERSION
+from fake_useragent.get_version import __version__ as VERSION
```

Comment:
I think `__version__` is more standard, so I wouldn't do this rename.

Reply:
What do you exactly mean?

I just renamed fake_useragent.settings to fake_useragent.get_version. Since the only thing that was left in the settings.py was version retrieval. So I renamed the python file in the src directory.

I didn't do any other renaming here.

Reply:
Owh you mean I shouldn't do the rename to VERSION. but that is already used I believe on other places in the code. Let me check.

Reply:
I'm now conform PEP 8.